src/controllers/reportingController.ts

```diff
@@ -47,7 +46,6 @@ export class ReportingController {
 					},
 					organisationListForTypeAhead: organisationChoices.typeaheadOrganisations,
 					showTypeaheadOption: userCanAccessMultipleOrganisations,
-					backButton: '/reporting'
```

Comment:
If this line is deleted, the comma to the end of the previous line may need to be removed.

Reply:
This is OK by Typescript standards (it's even required when using strict linting)